### PR TITLE
Rework after successful Post Modal

### LINKED ISSUE #636: Skip button in a post enable the control flow to originating page
Current functionality:
On clicking the skip button the user is redirected to login page. 
Actual functionality:
It must redirect to the originating page ie the feed.

image

### LINKED ISSUE #687: Post only the success message after posting
As a logged in user, I would like to get a clear confirmation message that my post was successful

Current scenario:
I get a message to create my profile after my post is successful. Even when my profile is created and I am logged in. 

Expected behaviour:
Display only post successful message. Please ensure that this works for both organization and individual profiles.

image

### PR DESCRIPTION
**Pull request by jyclam**
### What does this pull request aim to achieve? 💯

Rework after successful Post modal:
* remove create profile button
* re-route skip button to link to newly created post

closes #636 
closes #687 

Looking for some refactor advice. I believe the `CreatePost` component is unmounted when it actually gets to posting to `/api/post` and get back the post ID (where I call `setPostUrl`.) As a result, I'm getting this error in the console.

```
index.js:1 Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in a useEffect cleanup function.
    in Form (at TabForms.js:76)
    in div (created by TabPane)
    in TabPane (at TabForms.js:75)
    in div (created by TabContent)
    in TabContent (created by Tabs)
```
Should I move `setPostUrl` into the Context object, or is there a more idiomatic way to accomplish this?

### CODE REVIEW

**Comment by jyclam** [comment]
Thanks @idjevm but not quite ready for final touches critiques yet. Looking to refactor inner workings first before I polish off the aesthetics

**Comment by joshmorel** [comment]
Yeah I was thinking @robinv85 . I'm still to new to react 16, hooks, context, etc to figure this out quickly.

**Comment by jyclam** [comment]
So, I've discovered that the errors weren't actually introduce in my changes. I will create a separate issue for it. PR is ready for full review now.

File: client/src/components/CreatePost/CreatePost.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
 import React, { useState, createContext, useContext } from "react";
 import { Row, Col } from "antd";
+
```
Comment (idjevm) [inline client/src/components/CreatePost/CreatePost.js:3]: remove this line please!

File: client/src/components/CreatePost/CreatePost.js [hunk 2]
```diff
@@ -127,15 +130,18 @@ const Wrapper = ({ onCancel, visible, children }) => {
...
   return (
     currentStep === 4 && (
       <>
         <TitleStep fontSize={typography.size.xlarge} currentStep={currentStep}>
           Success
         </TitleStep>
-        <CreateProfileButton primary>Create Profile</CreateProfileButton>
-        <CreateOrgLink to={""}>Skip</CreateOrgLink>
+
```
Comment (idjevm) [inline client/src/components/CreatePost/CreatePost.js:141]: remove extra line :)

**Comment by ppagarawal** [comment]
> @Gtstride So yeah we are not using the complete profile anymore. We will give the option to the user to go and see their newly created profile or to click close on the modal and be back on the feed. At least for soft launch, this should be fine right @ppagarawal ? 

*The user has the option to check the newly created post and to click close on the modal and be back on the feed

**Pull request merged**